bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -684,6 +710,35 @@ export default {
       })
       toast.open()
       return toast
+    },
+    purgeServiceWorkerAndCaches () {
+      navigator.serviceWorker.getRegistrations().then(function (registrations) {
+        for (let registration of registrations) {
+          registration.unregister().then(function () {
+            return self.clients.matchAll()
+          }).then(function (clients) {
+            clients.forEach(client => {
+              if (client.url && 'navigate' in client) {
+                setTimeout(() => { client.navigate(client.url.split('#')[0]) }, 1000)
+              }
+            })
+          })
+        }
+      })
+      window.caches.keys().then(function (cachesNames) {
```

Comment:
In about.vue there's some logic to only show the button when it's relevant:

https://github.com/openhab/openhab-webui/blob/013b23fc70c367900c1b874b16fb7df3f3b462e5/bundles/org.openhab.ui/web/src/pages/about.vue#L162-L175

I thought it was better not to show it if you don't have a service worker registered or cached assets which are only available if you access the UI on a secure origin; otherwise I think it will just confuse the user as the button will do nothing (and incidentally this code will break because `navigator.serviceWorker` and `window.caches` won't be defined). This is the most probable situation when users access OH on HTTP port 8080 in a local network and this option makes no sense in this situation.

Even if you access OH on HTTPS with port 8443 it still wouldn't work if the certificate isn't trusted. The other option is to force your OH URL as a secure location manually with e.g. [chrome://flags/#unsafely-treat-insecure-origin-as-secure](chrome://flags/#unsafely-treat-insecure-origin-as-secure), which is obviously not formally recommended.